### ISSUE TITLE
All poll reaction options are set as having 1 vote from the person who added the integration.

### ISSUE THREAD

**Issue by ghost** [xoneco/simple-poll#21]
Would prefer this to be from @slackbot or @simplepoll that way other people can use the poll and I can vote on the ones they make.

**Comment by ghost** [comment]
I have the same problem. Why is there one vote in each options to begin with? Shouldn't it be 0? I am confused.

**Comment by wilhelmklopp** [comment]
Hey,
Yes, this is a known problem that has been around for a while. 
This will improve as soon as Slack adds the necessary functionality to their API, which is soon!

Duplicate of #3

**Comment by ghost** [comment]
@xoneco It seems the ability to add a new bot and have it add voting options already exists with Polly (www.polly.ai) + Lunchbox (http://lunchbox.352inc.com/). This would be make SimplePoll more functional while you wait on the Slack API update to not need a new user created.

**Comment by wilhelmklopp** [comment]
Yup, theoretically this would be possible with a bot user.

The link which explains the reasoning behind not using a bot user is also in #3 

Basically:
1. The bot would have to be in any channel where you want to create a poll (in order to add reactions), thus before you create any poll you will first have to invite the bot to a channel. This partly goes against the "simple" part of Simple Poll.
2. (and this is a big one) There are still a lot of privacy concerns around bots. A bot receives any and all messages in a given channel. This is something that a lot of teams **do not** want.

**Comment by wilhelmklopp** [comment]
Oh and regarding the suggestion that @slackbot should be able to do reactions as well, that would be very very cool, I agree :+1:

**Comment by ghost** [comment]
Cool, love that you are thinking about user privacy as well. Regarding issue #1 does this also mean that the authorizing user has to be in every channel that SimplePoll is used in for it to work properly? 

If the authorizing user does not have to be in every channel for SimplePoll to work then couldn't teams just create a new user and authorize SimplePoll with that account? Still not exactly as 'simple' as a one-click authorization, but an easily manageable work-around nonetheless.

**Comment by wilhelmklopp** [comment]
In that case the user wouldn't have to be in any of the channels, but the bot would have to be all of the channels where you'd want to create polls.

**Comment by ghost** [comment]
Maybe I missed it but this should be explained on the front page of this plugin. To me it basically makes simple-poll unusable:
a) the votes are always off by one
b) the creator of the poll cannot really vote themselves.

**Comment by wilhelmklopp** [comment]
@cookierebel It's also explained in the FAQ: https://simplepoll.rocks/faq/

The emoji reaction initiation is only there for convenience of course. You can **always** add/remove emoji reactions yourself (as you would on any other message in Slack) and only vote on your desired options. 

So for example once other people have voted, you (as the poll creator) can un-vote on the options you do not want to pick. Then the vote count is accurate (a) and the poll creator can vote (b).

But of course it is not ideal. Improvements hopefully coming soon™

**Comment by DavidSiska** [comment]
Hi @xoneco, the issue is still there :-( Any idea when it will be fixed?

**Comment by wilhelmklopp** [comment]
@DavidSiska This should be fixed now. If you enable button polls in the settings: https://simplepoll.rocks/dashboard/ it should work, because it's using buttons instead of reactions.
